app/scripts/modules/kubernetes/loadBalancer/details/details.html

```diff
@@ -171,5 +171,11 @@ <h3 select-on-dbl-click>
         </kubernetes-key-value-details>
       </dl>
     </collapsible-section>
+    <collapsible-section heading="Service Labels" ng-if="loadBalancer.description.serviceLabels">
```

Comment:
Can I get in a situation where `loadBalancer.description.serviceLabels` is an empty object (in that case this section would be rendered but empty)?

Reply:
Not sure tbh, but everything works the same as with the annotations so the behavior would be consistent ;)

Reply:
Is there anything else you expect from me?